bin/copyRawDataToPrm.sh

```diff
@@ -339,15 +350,16 @@ function splitSamplesheetPerProject() {
 	_allProjects="${_allProjects// /,}"
 	printf '%s\n' "\"${_allProjects}\"" > "${JOB_CONTROLE_FILE_BASE}.trace_putFromFile_overview.csv" 
 	#
-	# Move samplesheet to archive on sourceServerFQDN
+	# remove samplesheet to archive on sourceServerFQDN
```

Comment:
waaron verwijder je de samplesheet?

Reply:
Een samplesheet in een archive map triggert to niks? Wel een archive hebben betekent dat je nog eens iets uit dat archief kunt vissen om het opnieuw te doen als het de eerste keer niet goed was. Nu moet je de samplesheet ergens anders vandaan gaan halen. Indien het een remove blijft dan wel graag het comment aanpasen want dat klopt nu niet en is erg verwarrend.